src/style.css

```diff
@@ -0,0 +1,36 @@
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
```

Comment:
remove this background, you already have this style below

Reply:
below style active